### ISSUE TITLE
Destroying a created chart

### ISSUE THREAD

**Issue by Quas94** [chartist-js/chartist#541]
Once I've created a Chartist chart and I have a reference to it, is there any way to destroy it such that it's just removed from the page (without CSS gimmicks like changing visibility, display type, etc)?

The Chartist API is somewhat confusing and I'm not sure which functions can be called from which types.

Thanks

**Comment by gionkunz** [comment]
Call `chart.detach()` and empty the chart container element.

**Comment by Quas94** [comment]
Ahhh okay. I called detach but didn't think to clear the container.

Worked - thanks!

**Comment by gionkunz** [comment]
Cool, glad it's working out for you.
